README.md

```diff
@@ -272,6 +322,30 @@ on_success:
     chat_message_file: the-message-dir/msg.txt
 ```
 
+If using send to chat only and the `chat_message_file` parameter, the put step requires only one ["put inputs"]. For example:
+
+```yaml
+on_success:
+  put: gh-status
+  # the-message-dir: "output" of a previous task
+  inputs: [the-message-dir]
+  params:
+    state: success
+    chat_message_file: the-message-dir/msg.txt
+```
+
+If using send to chat only without `chat_message_file` parameter, `inputs` may be left empty.
+
+```yaml
+on_success:
+  put: gh-status
+  # It could be omitted without any error, but better to specify an empty list (see note below).
+  inputs: []
+  params:
+    state: success
```

Comment:
same question here: why do we need state for gchat only?

Reply:
As before, state is optional and as for GitHub status, it may be activated only for state success.